opennms-webapp/src/main/webapp/heatmap/heatmap-box.jsp

```diff
@@ -118,80 +118,82 @@
     <div id="treemap"></div>
 
     <script type="text/javascript">
-        require(['jquery', 'jquery-ui/jquery-ui', '../js/jquery.ui.treemap'], function( $ ) {
-            var mouseclickHandler = function (e, data) {
-                var nodes = data.nodes;
-                var ids = data.ids;
-                <%
-                  if ("foreignSources".equals(heatmap)) {
-                %>
-                location.href = "<%=request.getRequestURI()%>?mode=<%=mode%>&heatmap=nodesByForeignSource&foreignSource=" + nodes[0].id;
-                <%
-                  }
-
-                  if ("categories".equals(heatmap)) {
-                %>
-                location.href = "<%=request.getRequestURI()%>?mode=<%=mode%>&heatmap=nodesByCategory&category=" + nodes[0].id;
-                <%
-                  }
-
-                  if ("monitoredServices".equals(heatmap)) {
-                %>
-                location.href = "<%=request.getRequestURI()%>?mode=<%=mode%>&heatmap=nodesByMonitoredService&monitoredService=" + nodes[0].id;
-                <%
-                  }
+        window.onload = function() {
```

Comment:
This may break any other scripts on the page that rely on `window.onload`, since only a single function can be set.

If we leverage `window.onload`, we should make sure that any other function which was previously assigned is also called.